fast_pred/src/fastlightfm.cpp

```diff
@@ -190,15 +191,28 @@ void FastLightFM::load(string dir) {
   user_features = CSRMatrix::newInstance(cnpy::npz_load(dir + "/user-features.npz"));
   item_features = CSRMatrix::newInstance(cnpy::npz_load(dir + "/item-features.npz"));
 
+  init();
+}
+
+void FastLightFM::init() {
+
   assert(item_embeddings.shape[1] == user_embeddings.shape[1]);
   no_components = item_embeddings.shape[1];
+
+  number_of_items = item_features->rows;
```

Comment:
I thought we will filter some of the items when calling the predict , so it shouldn't be in init?